### DISCUSSION TITLE
Bring back the Cakefile

### DISCUSSION THREAD

**Discussion by aslilac** [DanielXMoore/Civet#1084] [Ideas]
Cakefile's and the `cake` CLI were one of my favorite things about CoffeeScript, and Civet just happens to have a name that also starts with C. What would you think of reclaiming `cake`? :)

**Comment by STRd6** [comment]
These days I'd probably recommend https://github.com/casey/just or adding scripts to `package.json`. But if anyone wants to remake `cake` in Civet I'd add it to my list of recommendations 😸

**Comment by edemaine** [comment]
I've never used `cake` but https://coffeescript.org/v2/annotated-source/cake.html is impressively simple. It'd certainly be possible to make a separate CLI tool/NPM package for this (`civet-cake` or some more clever combination?) that instead loaded using Civet. Maybe it'd be better to allow `Cakefile`, `Cakefile.coffee`, or `Cakefile.civet` so you can specify the language? Could be nice for backward compatibility with existing CoffeeScript projects that want to migrate to Civet.
